addon/components/tooltip-on-element.js

```diff
@@ -67,8 +67,11 @@ export default EmberTetherComponent.extend({
   'aria-hidden': computed.not('tooltipIsVisible'),
   attributeBindings: ['aria-hidden', 'role', 'tabindex'],
   classNameBindings: ['effectClass'],
-  classPrefix: 'tooltip',
-  classNames: ['tooltip'],
+  classPrefix: 'ember-tooltip',
+  classNames: ['ember-tooltip', 'tooltip'],
```

Comment:
Come to think of it I don't think we should keep the `.tooltip` class. The CSS can mix in two ways.
* Using `.ember-tooltip-*` makes sure that this library's CSS doesn't affect the project (ie. Zenefits app) code.
* But if we keep `.tooltip` here then a projects' tooltip classes may unintentionally affect the style of this library's tooltips.

I agree that we should make a clean break and only use `.ember-tooltip-*` classes. We should add a release note saying that if people want to add additional style then they need to apply it to the `.ember-tooltips` class instead of `.tooltip`.

I'll make a new commit that addresses this, run the tests again, then I believe we're good to go.